frontend/cypress/pages/metrics/metrics.ts

```diff
@@ -1,5 +1,17 @@
 class Metrics {
-  private readonly cycleTimeSettingTitle = () => cy.contains('Cycle time settings').should('exist')
+  private readonly crewsSettingTitle = () => cy.contains('Crews setting')
+  private readonly crewsSettingContent = () => cy.contains('Yu Zhang')
+  private readonly cycleTimeSettingTitle = () => cy.contains('Cycle time settings')
+  private readonly importCycleTimeSettingContent = () => {
+    cy.contains('Analysis').should('exist')
```

Comment:
Learn `how to wriete E2E test case with POM`

Reply:
I have rewrite e2e test for import a json file